api/admin/problem_details.py

```diff
@@ -168,15 +168,6 @@
     detail=_("You must identify the analytics service by its name."),
 )
 
-COLLECTION_NAME_ALREADY_IN_USE = pd(
```

Comment:
This has been replaced with the more generic `INTEGRATION_NAME_ALREADY_IN_USE` problem detail, since collections take their names from integrations now.